internal/agent/hooks/kcrypt_uki.go

```diff
@@ -130,6 +130,10 @@ func (k KcryptUKI) Run(c config.Config, _ v1.Spec) error {
 			if part == "" {
 				c.Logger.Infof("Partition %s not found, waiting %d seconds before retrying", p, i)
 				time.Sleep(time.Duration(i) * time.Second)
+				// Retry the unlock as well, because maybe the partition was not refreshed on time for unlock to unlock it
+				// So no matter how many tries we do, it will still be locked and will never appear
+				err := kcrypt.UnlockAll(true)
+				c.Logger.Debugf("UnlockAll returned: %s", err)
```

Comment:
should we maybe add this Debug message also in the first UnlockAll, to see if the initial one already locked everything

Reply:
this is actually wrong, we should check if the err is not nil before printing absurd empty stuff lol

This line was autosuggested by copilot, thats what you get for not reviewing stuff arg